File: Code Chef/JavaScript/300-399/390 Spice Level KITCHENSPICE.js
process.stdin.setEncoding("utf-8");

const strToInt = (strArray) => {
    const intArray = strArray.map(Number);
    return intArray;
};

const categorizeBySpiceLevel = (spiceLevel) => {
    (spiceLevel < 4)
        ? console.log("MILD")
        : ((spiceLevel >= 4) && (spiceLevel < 7))
            ? console.log("MEDIUM")
            : console.log("HOT");
};

process.stdin.on("data", (input) => {
    const testcases = strToInt(input.trim().split("\n").slice(1));
    
    testcases.forEach((spiceLevel) => {
        categorizeBySpiceLevel(spiceLevel);
    });
});

File: Code Chef/C Programming/300-399/390 Spice Level KITCHENSPICE.c
#include <stdio.h>
int main(void)
{
	int t,x;
	scanf("%d",&t);
	for(int i=0;i<t;i++)
	{
	    scanf("%d",&x);
	    if(x<4)
	    {
	        printf("MILD\n");
	    }
	    else if((x>=4) && (x<7))
	    {
	        printf("MEDIUM\n");
	    }
	    else if(x>=7)
	    {
	        printf("HOT\n");
	    }
	}
	return 0;
}
